lib/Backend/Commands.hs

```diff
@@ -266,6 +268,7 @@ copyCmd (CopyOptions oldPath newPath force) = do
       Right dir -> copyDir dir
 
   -- Validation checks.
+  checkForEntriesInPath newPath
```

Comment:
Given this tree:
```
$ cabal run exe:coffer -- create test/dir/entry1                                        
$ cabal run exe:coffer -- create test/dir/entry2
$ cabal run exe:coffer -- view /

/
  test/
    dir/
      entry1 - [2022-02-24 11:25:51]
      entry2 - [2022-02-24 11:25:53]
```

Previously, when you tried to copy `entry1` on top of `entry2`, you'd get this message:

```
$ cabal run exe:coffer -- copy test/dir/entry1 test/dir/entry2

[ERROR] The following entries cannot be copied because an entry already exists at the destination.
Use '--force' or '-f' to overwrite existing entries.

Cannot copy '/test/dir/entry1' to '/test/dir/entry2'.
```

But now, after these changes, the same command returns:
```
[ERROR] The following entries cannot be copied because a destination directory has an entry '/test/dir/entry2' in it's subpath.
```

And even with `-f`, the entry cannot be copied.

Reply:
Done